### PR TITLE
General stability patches from netatalk-classic etc.

### PR DESCRIPTION
**Pull request by rdmark**
Patches from the 20200806 release tag: https://github.com/christopherkobayashi/netatalk-classic/releases/tag/20200806

### CODE REVIEW

File: etc/afpd/status.c [hunk 1]
```diff
@@ -671,7 +671,11 @@ void set_signature(struct afp_options *options) {
                     goto server_signature_random;
                 } else {
                     fseek(fp, -1L, SEEK_END);
-                    if(fgetc(fp) != '\n') fputc('\n', fp); /* last char is \n? */
+                    if(fgetc(fp) != '\n')
+                    {
+                        fflush(fp);
+                        fputc('\n', fp); /* last char is \n? */
```
Comment (slowfranklin) [inline etc/afpd/status.c:677]: Why? This doesn't seem to make sense.

Comment (rdmark) [reply etc/afpd/status.c:677]: As I understand it, this is in order to flush the bidirectional i/o stream when switching direction.

Comment (rdmark) [reply etc/afpd/status.c:678]: I had a chat with Chris. We would suggest using fseek here instead for C11 conformance. Please me know if this looks good to you now. I'll make a clean PR.

Comment (slowfranklin) [reply etc/afpd/status.c:678]: Ah, that one, now I remember. So either please squash the two commits or drop the last one.

Comment (rdmark) [reply etc/afpd/status.c:678]: New PR in https://github.com/Netatalk/Netatalk/pull/168

**Pull request closed without merging**